CHANGELOG.md

```
@@ -4,10 +4,18 @@ All notable changes to this project will be documented in this file. The format
 ## [Unreleased]
 ### Added
-- Nothing yet.
+- Builders de ranking para buscas textuais: `Match.Score` no MySQL e `TsVector.RankWebSearch/RankPlainQuery` no PostgreSQL, incluindo ordenação por relevância e normalização opcional.
+- Roadmap expandido com novos próximos passos para evolução do pacote.
+- Alternância de idioma/configuração para buscas textuais PostgreSQL via `TsVector.WithLanguage` (compatível com `WithConfig`).
+- Aliases automáticos para subconsultas em `FROM`/`JOIN`, garantindo SQL válido em dialetos que exigem nomeação.
+- Helpers parametrizados para JSON/JSONB (`JSONExtract`, `JSONExtractText`, `JSONContains`) compatíveis com MySQL e PostgreSQL.
+- Configuração global do idioma padrão para buscas textuais PostgreSQL via `SetDefaultTextSearchConfig`.
+- Combinação de consultas com `UNION`/`UNION ALL` e ordenação/paginação finais.
 
 ### Changed
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)